genie-common-internal/src/test/groovy/com/netflix/genie/common/internal/dto/v4/JobRequestSpec.groovy

```diff
@@ -433,6 +440,7 @@ class JobRequestSpec extends Specification {
                 .build()
         def requestedAgentConfig = new AgentConfigRequest.Builder()
                 .withArchivingDisabled(archivingDisabled)
+                .withRequestedArchiveLocationPrefix(requestedArchiveLocationPrefix)
                 .withTimeoutRequested(timeout)
                 .withInteractive(interactive)
                 .withRequestedJobDirectoryLocation(jobDirectoryLocation)
```

Comment:
I suggest at least one test case where it's null (it's going to surface the bugs in conversion i pointed out above)

Reply:
Sure